packages/graphiql-react/.gitignore

```diff
@@ -24,4 +25,8 @@ types
 *.sln
 *.sw?
 
+# generated files
+vite.config.js
+vite.config.d.ts
```

Comment:
can you remove also generated files `vite.config.js`, `vite.config.d.ts`  from `packages/graphiql-react`?